encoder/vaapiencoder_h264.cpp

```diff
@@ -538,8 +538,12 @@ class VaapiEncStreamHeaderH264
             m_headers.insert(m_headers.end(), s, e);
             if (e == h.end())
                 break;
-            m_headers.insert(m_headers.end(), emulation, emulation + N_ELEMENTS(emulation));
+            
             s = e + N_ELEMENTS(zeros);
+            if (*s <= 3)
```

Comment:
have some comments here, please. why it's 3 not 4, 5, 6....not every code ready will do "git blame" and see commit log.